api/v1/helpers/verbs/utils.js

```diff
@@ -565,7 +565,7 @@ function cleanAndStripNulls(obj) {
 
       // if undefined, parentAbsolutePath needs to be set to empty string,
       // to pass swagger's schema validation
-      if (key === 'parentAbsolutePath' && !o[key]) {
+      if ((key === 'parentAbsolutePath' || key === 'createdBy') && !o[key]) {
```

Comment:
don't we always want to have a createdBy field in the response, even if the value is ''?